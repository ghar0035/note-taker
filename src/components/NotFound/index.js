import { Link } from "react-router-dom";
import img404 from '../../images/404.png'
import { Container } from "../Register";

const NotFound = ({ user }) => {
   return (
    <Container className="container">
        <img src={img404} alt="not found" width={600} />

        <div>
            <Link to={ user ?   '/dashboard' : '/' } >
            Take me to  { user ?   'dashboard' : 'home' }
            </Link>
        </div>
    </Container>
   )
}

export default NotFound;
