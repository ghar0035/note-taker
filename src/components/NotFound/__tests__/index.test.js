import renderer from "react-test-renderer";
import NotFound from "../index.js";
import { MemoryRouter as Router } from "react-router-dom";

test("renders NotFound", () => {
    const tree = renderer
      .create(
        <Router>
          <NotFound />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });