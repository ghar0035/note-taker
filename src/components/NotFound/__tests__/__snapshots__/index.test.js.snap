// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound 1`] = `
<div
  className="sc-gswNZR kNyIcL container"
>
  <img
    alt="not found"
    src="404.png"
    width={600}
  />
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      Take me to  
      home
    </a>
  </div>
</div>
`;
